app/services/rabbitmq/RabbitMQMessageService.scala

```diff
@@ -233,8 +233,6 @@ class RabbitMQMessageService extends MessageService {
     // This probably isn't going to extract queue (use other submit() for that) so make a new broker
     val tempChannel = connection.get.createChannel()
     tempChannel.exchangeDeclare(exchange, exchange_type, true)
```

Comment:
```suggestion
    tempChannel.exchangeDeclare(exchange, exchange_type, true)
    
    // If a routing_key (queue name) was provided, ensure that the queue exists
    if (routing_key != "") {
      tempChannel.queueDeclare(routing_key, true, false, false, null)
      tempChannel.queueBind(routing_key, exchange, routing_key)
    }
```

I think we should either remove the `routing_key` parameter for this method, or properly handle both cases here. If a user passes an empty routing_key, we send to the exchange. Otherwise, we make sure the target queue exists before publishing. This way, we can change the default to "" in `EventSinkService` if we prefer the exchange-based approach, while still preserving functionality for a non-exchange based approach to sending RMQ messages.